Plutarch/Api/V1/Value.hs

```diff
@@ -286,14 +408,14 @@ plovelaceValueOf = phoistAcyclic $
 punionWith ::
   Term
     s
-    ( (PInteger :--> PInteger :--> PInteger) :--> PValue 'Sorted _ :--> PValue 'Sorted _
+    ( (PInteger :--> PInteger :--> PInteger) :--> PValue 'Sorted any0 :--> PValue 'Sorted any1
```

Comment:
I allow different `AmountGuarantees` in the binary `PValue` functions in a lot of places since technically it should be valid.